.github/workflows/ci.yml

```diff
@@ -818,7 +838,7 @@ jobs:
   done:
     name: CI Finished
     runs-on: ubuntu-latest
-    needs: [ package ]
+    needs: [ package, docker-build-test ]
```

Comment:
I since `docker-build-test` does not produce the musl binaries, it is not required for this step any more.